crates/holochain/benches/multi_install.rs

```diff
@@ -0,0 +1,426 @@
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use criterion::criterion_group;
+use criterion::criterion_main;
+use criterion::BenchmarkId;
+use criterion::Criterion;
+
+use holochain::sweettest::*;
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_conductor_api::AdminInterfaceConfig;
+use holochain_conductor_api::InterfaceDriver;
+use holochain_serialized_bytes::{SerializedBytes, UnsafeBytes};
+use holochain_types::prelude::InstallAppBundlePayload;
+use holochain_types::prelude::{AppBundleSource, DnaFile};
+use kitsune_p2p::KitsuneP2pConfig;
+use kitsune_p2p_types::config::tuning_params_struct::KitsuneP2pTuningParams;
+use reqwest::Client;
+use tokio::runtime::Builder;
+use tokio::runtime::Runtime;
+
+criterion_group!(benches, multi_install);
+
+criterion_main!(benches);
+
+struct Settings {
+    /// Number of separate machines you want to run the test on.
```

Comment:
what if a value >1 is provided here and the bootstrap server only knows one machine?